### PR TITLE
Implementation of #155

### PR DESCRIPTION
**Pull request by IremToroslu**
#155 :)

### CODE REVIEW

**Review by SaiVarunVaranasi — COMMENTED** [review]
Looks Good to me:-)

File: frontend/src/components/details/TableComponent.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ class TableComponent extends Component {
     render() {
         const idKey = this.props.tableKey;
         return (
-            <Container fluid={true}>
+            <Container fluid={true} className='TableCorner'>
```
Comment (Waldleufer) [inline frontend/src/components/details/TableComponent.jsx:19]: This name is not matching with the css name. I will fix that shortly.

File: frontend/src/components/details/charts_tables.css [hunk 2]
```diff
@@ -101,13 +102,18 @@
     margin-top: 10px;
     margin-bottom: 10px;
 }
+.tableCorner{
```
Comment (Waldleufer) [inline frontend/src/components/details/charts_tables.css:105]: this one doesn't match.

File: frontend/src/components/mydashboard/MydashboardItemComponent.js [hunk 3]
```diff
@@ -2,12 +2,19 @@ import React from 'react';
...
+// const data = [];
 
-const data = [];
-
-for (let x = 1; x <= 24; x++) {
-    data.push({ x: x, y: Math.floor(Math.random() * 100) });
-}
+// for (let x = 1; x <= 24; x++) {
+//     data.push({ x: x, y: Math.floor(Math.random() * 100) });
+// }
```
Comment (Waldleufer) [inline frontend/src/components/mydashboard/MydashboardItemComponent.js:17]: this can be removed.

**Review by Waldleufer — COMMENTED** [review]
Looking good. In my Opinion the wide margins don't look as good as the slim margins before, but I guess that's a matter of taste 🙂

File: frontend/src/components/details/TableComponent.jsx (hunk 1 above)
Comment (IremToroslu) [reply frontend/src/components/details/TableComponent.jsx:19]: Thanks @Waldleufer 😄 I probably missed that one somehow

**Review by Waldleufer — COMMENTED** [review]
🙂

File: frontend/src/components/details/TableComponent.jsx (hunk 1 above)
Comment (Waldleufer) [reply frontend/src/components/details/TableComponent.jsx:19]: done. But it does not affect the table. the corners still are sharp ._.

File: frontend/src/components/details/DetailsComponent.js [hunk 4]
```diff
@@ -127,12 +127,27 @@ class DetailsComponent extends Component {
...
 
+        if (
+            this.props.selectedProduct.productID === undefined ||
+            this.props.selectedProduct.productID === ''
+        ) {
+            return (
+                <Container className='ScenarioContainer' id='capture' fluid>
+                    <LoadingComponent
+                        loading
+                        text={'Please go to Categories and select something :)'}
```
Comment (SaiVarunVaranasi) [inline frontend/src/components/details/DetailsComponent.js:138]: Cool:-)

**Review by SaiVarunVaranasi — APPROVED** [review]

File: frontend/src/components/sidebar/SidebarComponent.js [hunk 5]
```diff
@@ -22,13 +22,12 @@ const useStyles = createUseStyles({
...
  * The SidebarComponent consists out of the functionality and the Look and Feel of the Left-Side-Navigationbar
  *
- * @author Irem Toroslu, Martin Wagner
  */
 function SidebarComponent({ pageWrapId, outerContainerId }) {
     const { push } = useHistory();
     const theme = useTheme();
     const classes = useStyles({ theme });
-    const isMobile = window.innerWidth <= 1080;
+    const isMobile = false;
```
Comment (MartinDuersch) [inline frontend/src/components/sidebar/SidebarComponent.js:30]: 😄

**Review by MartinDuersch — APPROVED** [review]
Improvement in look and feel

**Pull request merged**